### PR TITLE
Quick and dirty mixin to time and log event timings to a "csv" (uses semicolon)

### PR DESCRIPTION
**Pull request by mitchej123**
Adds an optional mixin to log event timing to a csv

### CODE REVIEW

File: src/main/java/com/mitchej123/hodgepodge/mixins/preinit/MixinLoadController_logModTimes.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import cpw.mods.fml.common.LoadController;
+import cpw.mods.fml.common.LoaderState;
+import cpw.mods.fml.common.ModContainer;
+import cpw.mods.fml.common.event.FMLEvent;
+import it.unimi.dsi.fastutil.objects.Object2ObjectOpenHashMap;
+
+@Mixin(value = LoadController.class, remap = false)
+public class MixinLoadController_logModTimes {
+
+    private static Map<String, ArrayList<ModProfileResult>> results = new Object2ObjectOpenHashMap<>();
```
[the remark this answers is not in the document]
Comment (mitchej123) [reply src/main/java/com/mitchej123/hodgepodge/mixins/preinit/MixinLoadController_logModTimes.java:29]: The results end up sorted which is why I went with an ArrayList

[the remark this answers is not in the document]
Comment (glowredman) [reply src/main/java/com/mitchej123/hodgepodge/mixins/preinit/MixinLoadController_logModTimes.java:29]: Makes sense.

**Review by glowredman — APPROVED** [review]
lgtm

**Comment by boubou19** [comment]
For reference: use `-Dhodgepodge.logModTimes=true` to enable it.

**Pull request merged**